zephyr/modules/owntech_hrtim_driver/zephyr/src/hrtim.c

```diff
@@ -1317,218 +1376,218 @@ void hrtim_change_frequency(uint32_t new_frequency)
-            (float32_t)LL_HRTIM_TIM_GetPeriod(                                \
-                                    HRTIM1, 
+            /
+            (float32_t)LL_HRTIM_TIM_GetPeriod(
+                                    HRTIM1,
                                     LL_HRTIM_TIMER_B);
 
-        phase_shift = 
-            (float32_t)LL_HRTIM_TIM_GetCompare2(                              \
-                                    HRTIM1,                                   \
+        phase_shift =
+            (float32_t)LL_HRTIM_TIM_GetCompare2(
+                                    HRTIM1,
                                     LL_HRTIM_TIMER_A)
-            /(float32_t)LL_HRTIM_TIM_GetPeriod(                               \
-                                    HRTIM1,                                   \
+            /(float32_t)LL_HRTIM_TIM_GetPeriod(
+                                    HRTIM1,
                                     LL_HRTIM_TIMER_B);
 
         if(timerB.pwm_conf.modulation == UpDwn)
         {
             LL_HRTIM_TIM_SetPeriod(HRTIM1, LL_HRTIM_TIMER_B, period/2);
-            LL_HRTIM_TIM_SetCompare1(HRTIM1,                                  \
-                                     LL_HRTIM_TIMER_B,                        \
+            LL_HRTIM_TIM_SetCompare1(HRTIM1,
+                                     LL_HRTIM_TIMER_B,
                                      duty_cycle*period/2);
 
             hrtim_phase_shift_set(PWMB, phase_shift*period/2);
             tu_channel[PWMB]->pwm_conf.period = period/2;
         }
-        else 
+        else
         {
             LL_HRTIM_TIM_SetPeriod(HRTIM1, LL_HRTIM_TIMER_B, period);
-            LL_HRTIM_TIM_SetCompare1(HRTIM1,                                  \
-                                     LL_HRTIM_TIMER_B,                        \
+            LL_HRTIM_TIM_SetCompare1(HRTIM1,
+                                     LL_HRTIM_TIMER_B,
                                      duty_cycle*period);
 
             hrtim_phase_shift_set(PWMB, phase_shift*period);
             tu_channel[PWMB]->pwm_conf.period = period;
         }
 
-        duty_cycle = 
-            (float32_t)LL_HRTIM_TIM_GetCompare1(                              \
-                                    HRTIM1,                                   \
-                                    LL_HRTIM_TIMER_C)                         \
-            /                                                                 \
-            (float32_t)LL_HRTIM_TIM_GetPeriod(                                \
-                                    HRTIM1,                                   \
+        duty_cycle =
+            (float32_t)LL_HRTIM_TIM_GetCompare1(
+                                    HRTIM1,
+                                    LL_HRTIM_TIMER_C)
+            /
+            (float32_t)LL_HRTIM_TIM_GetPeriod(
+                                    HRTIM1,
                                     LL_HRTIM_TIMER_C);
```

Comment:
This has poor readability: we should store the results of both GetPeriod calls in two variables, then do the division on a single line.